src/Query.php

```diff
@@ -311,19 +311,19 @@ public function where($conditions, $overwrite = false)
     }
 
     /**
-     * Sets the main query to use. Queries added using this method
+     * Modifies the query part, taking scores in account. Queries added using this method
      * will be stacked on a bool query and applied to the must part of the final BoolQuery.
      *
      * This method can be used in the same way the `where()` method is used. Please refer to
      * its documentation for more details.
      *
-     * @param array|callable|\Elastica\Filter\AbstractFilter $conditions The lsit of conditions
+     * @param array|callable|\Elastica\Filter\AbstractFilter $conditions The list of conditions
      * @param bool $overwrite Whether or not to replace previous queries.
      * @return Query
      */
     public function search($conditions, $overwrite = false)
```

Comment:
The only other name I can think of is `query()`. The [ElasticSearch docs refer to it as the 'query context'](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-filter-context.html).